test/test_dosemu.py

```diff
@@ -5116,6 +5128,8 @@ def test_pit_mode_2(self):
 
 PPDOSGITTestCase = ppdosgit(OurTestCase, {
     "test_comcom_r200fix_protected": UNSUPPORTED,
+    "test_command_com_cmdline_length_truncate01": UNSUPPORTED,
+    "test_command_com_cmdline_length_truncate02": UNSUPPORTED,
```

Comment:
So what's missing?

Reply:
Nothing, it's perfect. As far as I can see when presented with a extra long line you can either do:
1/ Copy a maximum of 126 chars to PSP, and set the length byte to 126 (<= MS-DOS)
or
2/ Copy a maximum of 126 chars to PSP, set the length byte to 127 and copy the whole command line to a en variable called CMDLINE (All our other test DOSes except OpenDOS 7.01).

One thing though, perhaps I should reduce the number of tests that do varying lengths below 126, probably two should suffice say 60 and 125 chars?

Reply:
Reducing is fine, there are quite
too long. But what is still UNSUPPORTED
with comcom64?